aws/internal/service/lakeformation/filter.go

```diff
@@ -0,0 +1,172 @@
+package lakeformation
```

Comment:
Is it possible to add unit tests for the functions in this file?

Reply:
Potentially.

Reply:
It would definitely help with long-term maintainability.

Reply:
You're definitely right. It's just how complicated the useful tests would be. I'm going to create a tracking issue so we can get this in.